randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -187,13 +193,17 @@ Extra - asset_id: 1317347388
   * Normal Door to Ice Ruins East/Door to Ice Ruins Access
   * Extra - dock_index: 0
   > Door to Phendrana Shorelines
-      Charge Beam or Missile or Plasma Beam
+      Any of the following:
+          Charge Beam or Shoot Plasma Beam
```

Comment:
again, just Charge Beam is enough here?

Reply:
I just pushed a bunch of missing requirements related to charge/combat.